Sources/XcodeMetadata/Providers/SystemFrameworkMetadataProvider.swift

```diff
@@ -68,6 +68,11 @@ public final class SystemFrameworkMetadataProvider: SystemFrameworkMetadataProvi
     }
 
     private func sdkPath(name: String, platform: Platform, type: SDKType, source: SDKSource) throws -> AbsolutePath {
+		if name == "XcodeKit.framework" && platform == .macOS {
+			let sdkRootPath = try AbsolutePath(validating: "/Library/Frameworks")
+			return sdkRootPath.appending(component: name)
+		}
```

Comment:
I'm pretty sure this is not the correct path. This needs to be prefixed with the `xcodeDeveloperSdkRootPath`